src/content/add-to-app/ios/add-flutter-screen.md

```diff
@@ -786,5 +786,5 @@ in any way you'd like, before presenting the Flutter UI using a
 [`SystemNavigator.pop()`]: {{site.api}}/flutter/services/SystemNavigator/pop.html
 [tree-shaken]: https://en.wikipedia.org/wiki/Tree_shaking
 [`WidgetsApp`]: {{site.api}}/flutter/widgets/WidgetsApp-class.html
-[`window.defaultRouteName`]: {{site.api}}/flutter/dart-ui/SingletonFlutterWindow/defaultRouteName.html
+[`PlatformDispatcher.defaultRouteName`]: {{site.api}}/flutter/dart-ui/PlatformDispatcher/defaultRouteName.html
```

Comment:
The `window` global is deprecated: https://docs.flutter.dev/release/breaking-changes/window-singleton